pom.xml

```diff
@@ -349,23 +349,11 @@
 <!-- DocumentPipeline end -->
 
 <!-- Elasticsearch begin -->
-  <dependency>
```

Comment:
Why are there pom changes in this PR?  please revert pom.xml

Reply:
pretty sure the current was out of spec not being generated ... but whatever, i reverted